pkg/operator/ceph/object/controller.go

```diff
@@ -333,27 +333,35 @@ func (r *ReconcileCephObjectStore) reconcileCreateObjectStore(cephObjectStore *c
 			}
 		}
 
+		objContext.Realm = realmName
+		objContext.ZoneGroup = zoneGroupName
+		objContext.Zone = zoneName
+		logger.Debugf("realm for object-store is %v, zone group for object-store is %v, zone for object-store is %v", objContext.Realm, objContext.ZoneGroup, objContext.Zone)
```

Comment:
```suggestion
		logger.Debugf("realm for object-store is %q, zone group for object-store is %q, zone for object-store is %q", objContext.Realm, objContext.ZoneGroup, objContext.Zone)
```

These are just strings right?